### ISSUE TITLE
Linux build cgo issues

### ISSUE THREAD

**Issue by SevereOverfl0w** [dizzyd/mcdex#56]
Running the Linux binary is getting me this:

```
~
❯ sha256sum /home/overfl0w/bin/mcdex
f069bc2ca88612e4698a8c2db25c92f0df33f07a00e65796443c87d9a2594c11  /home/overfl0w/bin/mcdex

~
❯ mcdex pack.create mypack 1.16.3  
2020/10/31 21:17:36 Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

**Comment by ngoetje** [comment]
I have the same problem.

**Comment by dizzyd** [comment]
@SevereOverfl0w @ngoetje - what distro/version of Linux are you using?

**Comment by DerEnderKeks** [comment]
I got the same issue. I tried it on Debian 10 in WSL2.

**Comment by dizzyd** [comment]
I've fixed the problem, I believe. I'm working on fixing some other bugs but will make a release later tonight. Thanks for the report!

**Comment by dizzyd** [comment]
Fixed in 0.10.3
